code/modules/hydroponics/beekeeping/beebox.dm

```diff
@@ -16,7 +16,7 @@
 
 
 /mob/living/carbon/human/bee_friendly()
-	if(dna && dna.species && dna.species.id == SPECIES_PODPERSON) //bees pollinate plants, duh.
+	if(dna && dna.species && dna.species.id == (SPECIES_PODPERSON || SPECIES_PODPERSON_WEAK)) //bees pollinate plants, duh.
```

Comment:
This code will not work.

The proper code should be
``if(dna && dna.species && (dna.species.id == SPECIES_PODPERSON || dna.species.id == SPECIES_PODPERSON_WEAK))``

Reply:
I'll make this change